jdaviz/core/template_mixin.py

```diff
@@ -126,6 +127,8 @@ def show_widget(widget, loc, title):  # pragma: no cover
         display(widget)
 
     elif loc.startswith('sidecar'):
+        if not get_kernel_id():
```

Comment:
@mariobuikhuizen - is this sufficient, or would accessing `get_base_url` be more appropriate (in which case can that be exposed back to python in ipypopout)?

Reply:
I don't think there is a 1 on 1 mapping of where ipypopout and sidecar work. I think sidecar has different requirements than ipypopout.

Reply:
do you have any suggestions on which conditions to use for each for now then?

Reply:
I don't know how to detect if sidecar is available.

Reply:
ok, then I think this is better than nothing for now and we can always revise the condition going forward 🤷‍♂️